documentation/modules/exploit/multi/http/motioneye_auth_rce_cve_2025_60787.md

```diff
@@ -0,0 +1,58 @@
+## Vulnerable Application
+
+This module exploits a template injection vulnerability in the MotionEye.
+
+MotionEye is vulnerable to OS Command Injection in configuration parameters such as image_file_name. 
+Unsanitized user input is written to Motion configuration files, 
+allowing remote authenticated attackers with admin access to achieve code execution when Motion is restarted.
+
+This vulnerability affects MotionEye versions <= 0.43.1b4 are vulnerable.
+Successful exploitation may result in the remote code execution under the privileges
+of the web server, potentially exposing sensitive data or disrupting survey operations.
+
+An attacker can execute arbitrary system commands in the context of the user running the web server.
+
+## Testing
+
+1. Use Docker to set up the MotionEye app
+
+`docker run -p 9999:8765 ghcr.io/motioneye-project/motioneye@sha256:718171663d28f04f2fb97244f8ef03a814367b06078fc00685acc47f61663890`
```

Comment:
Given this is meant to be unpatched, I'm curious why we are running a specific commit hash here. May want to double check if this is patched on latest version. See other comment about potential security patch